### DISCUSSION TITLE
Changing site colors

### DISCUSSION THREAD

**Discussion by LyrixCaz** [chrede88/qubt#86] [Q&A] [answered]
Hello, 
first and foremost love the theme! I'm new with hugo and I'm not sure if this is something I can adjust via some sort of toml or yaml file but would it be possible to adjust the colors? like i know the theme has a light/dark mode and that's cool too but can i change font colors, background, etc?

**Comment by chrede88** [comment, marked as answer]
The colors are hardcoded, so changing them is not easy. The theme is build using Tailwind which needs to be complied before any changes take effect. I understand that this would be a great feature, but it'll take some work on my part that I don't have time for at the moment.

**Reply by LyrixCaz** [reply]
kk, thanks for the quick response!

**Comment by Anatolii-Ovcharuk** [comment]
You also can look script in repository: https://github.com/Anatolii-Ovcharuk/JS-Module-Code
And look how it working here: https://anatolii-ovcharuk.github.io/JS-Module-Code/

**Comment by viogami** [comment]
i has used html in md files to get it🤗
